src/main/java/step3/ResultView.java

```diff
@@ -0,0 +1,18 @@
+package step3;
+
+public class ResultView {
+
+    public void printResult(Cars cars) {
+        System.out.println("실행 결과");
+        cars.getCars().forEach(car -> {
+            printCarPosition(car);
+            System.out.println();
+        });
+    }
+
+    private void printCarPosition(Car car) {
+        for(int i =0 ; i< car.getPosition(); i++) {
+            System.out.print("-");
+        }
```

Comment:
```suggestion
        for(int i = 0 ; i < car.getPosition(); i++) {
            System.out.print("-");
        }
```
코드 작성 시, 코드 컨벤션을 지켜주시는 것도 좋을 것 같습니다.